data/tokens.json

```diff
@@ -1693,12 +1695,6 @@
       "gtmds_button_background-color-disabled": {
         "value": "rgba({White.gtmds_button_background-color},0.4)",
         "type": "color"
-      },
-      "Primary": {
-        "gtmds_button_background-color-hover": {
-          "value": "rgba({White.gtmds_button_background-color}, 0.7)",
-          "type": "color"
-        }
```

Comment:
Why do you suggest to remove this? We would still need a style defined for hover on a white, primary button ...

Reply:
I don't remember why I removed this, probably by mistake. Let's revert this change.